### PR TITLE
mobilesalesapp-added user service

### PR DESCRIPTION
**Pull request by Bharathi-kannann**

### CODE REVIEW

File: services/user-service.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const dbPassword = "3589b77ff4cc367d60ae67e1f7dada03";
+const basicAuth = 'Basic ' + btoa(dbUsername + ':' + dbPassword);
+
+class UserService {
+
+    static login(email, password){
+
+        const url = "https://05025f1a-856b-47a0-aadb-52e737a386f3-bluemix.cloudantnosqldb.appdomain.cloud/mobilesalesapp_users/_find"
+
+const requestData = {
```
Comment (fresher-batch-2021) [inline services/user-service.js]: Format your code.

Comment (Bharathi-kannann) [reply services/user-service.js]: okay

Comment (Bharathi-kannann) [reply services/user-service.js]: fixed

File: services/user-service.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const dbUsername = "apikey-v2-1kdtmo28t5uulevcbb5m8mifmj5bd962vbuc18qwa0m4";
+const dbPassword = "3589b77ff4cc367d60ae67e1f7dada03";
+const basicAuth = 'Basic ' + btoa(dbUsername + ':' + dbPassword);
+
+class UserService {
+
```
Comment (fresher-batch-2021) [inline services/user-service.js:6]: Add comments.

File: services/user-service.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                email: email,
+                password: password
+            },
+            fields: ["_id", "name", "password"],
+        };
+
+        return axios.post(url, requestData, { headers: { Authorization: basicAuth } });
+
+    }
+    static register(registerValues) {
```
Comment (fresher-batch-2021) [inline services/user-service.js:34]: Add comments

**Review by Shal2849 — APPROVED** [review]

**Pull request merged**